src/v2/util/BaseLoginRouter.js

```diff
@@ -125,6 +125,16 @@ function (Okta, BrowserFeatures, Settings,
         this.defaultAuth();
       });
 
+      this.listenTo(this.appState, 'change:remediationSuccess', function (appState, trans) {
+        //set remediationSuccess
+        this.appState.set('remediationSuccess', trans);
```

Comment:
`change:remediationSuccess` means `remediationSuccess` had been change and why you set it again?

Reply:
I added this assuming that when we are moving forward in the pipeline and call BaseLoginModel which does `self.appState.trigger('remediationSuccess', trans)`, we will need a listener for this change ?

Reply:
`trigger('remediationSuccess'` wont go here because this for `change:remedationSuccess`.

Reply:
Ah, right I'll remove it for now. We have this listener in the BaseLoginController that is setting remediationSuccess.
```       
this.listenTo(this.options.appState, 'remediationSuccess', function (transaction) {
        this.options.appState.set('remediationSuccess', transaction);
      }); 
```